js/google_ima.js

```diff
@@ -1245,6 +1240,11 @@ require("../html5-common/js/utils/utils.js");
             if(ad.isLinear())
             {
               _linearAdIsPlaying = true;
+              if (this.savedVolume >= 0)
```

Comment:
Would this still work if you change the volume after the ad starts, with or without pausing it?